schema/resolvers.go

```diff
@@ -32,9 +32,8 @@ func ParentIDResolver(_ context.Context, _ ClientMeta, r *Resource, c Column) er
 	return r.Set(c.Name, r.Parent.ID())
 }
 
-// ParentResourceFieldResolver resolves a field from the parent's resource, the value is expected to be set
```

Comment:
This comment was confusing 🙃 If it's expected to be set, it should error out when it's not.